animata/card/card-spread.css

```
@@ -1,31 +1,186 @@
+
+  .card-spread-item:nth-child(3) {
+    @apply z-30;
+  }
+
+  .card-spread-item:nth-child(4) {
+    @apply z-40;
+  }
+
+  .card-spread-item__internal {
+    @apply origin-bottom motion-reduce:transition-none;
+    transition: transform 520ms var(--ease-gather);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)